packages/docusaurus-theme-classic/src/theme/DocPage/index.js

```diff
@@ -59,19 +45,15 @@ function DocPage(props) {
           <div className={styles.docSidebarContainer} role="complementary">
             <DocSidebar
               docsSidebars={docsSidebars}
-              path={isHomePage ? homePagePath : currentRoute.path}
+              path={currentRoute.path}
               sidebar={sidebar}
               sidebarCollapsible={sidebarCollapsible}
             />
           </div>
         )}
         <main className={styles.docMainContainer}>
           <MDXProvider components={MDXComponents}>
-            {isHomePage ? (
-              <DocItem content={content} />
-            ) : (
-              renderRoutes(baseRoute.routes)
-            )}
+            {renderRoutes(baseRoute.routes)}
```

Comment:
@lex111 was able to revert the changes that are no longer required for docs homepage support